tsconfig.build.json

```diff
@@ -6,7 +6,8 @@
     "lib": ["esnext", "es2018", "dom"],
     "declaration": true,
     "strict": true,
-    "skipDefaultLibCheck": true
```

Comment:
Astro brings in `@types/mdx`, which depends on the `JSX` namespace being defined. We can enable `skipLibCheck` here, but not in `tsconfig.json` apparently. Needs some solution to make the JSX namespace available.

Reply:
This file is now removed in https://github.com/vaadin/web-components/pull/9921

Reply:
Removed this change